openedx/core/djangoapps/waffle_utils/tests/test_init.py

```diff
@@ -72,9 +68,6 @@ def test_course_waffle_flag(self, data, mock_set_custom_attribute):
                     self.TEST_COURSE_KEY
                 )
 
-        self._assert_waffle_flag_attribute(mock_set_custom_attribute, expected_flag_value=str(data['result']))
-        mock_set_custom_attribute.reset_mock()
```

Comment:
I was thinking we could swap this for an assertion that _monitor_value is called. Can we add a follow-up task to restore that?

Reply:
At first I tried that, but it was really difficult to have a mock that works for both versions of edx-toggles.